SprawdzanieNIPu/ConsoleApplication1/Program.cs

```diff
@@ -1,17 +1,34 @@
-﻿using System;
-
+﻿//-----------------------------------------------------------------------
+// <copyright file="Program.cs" company="CompanyName">
+// Company copyright tag.
+// </copyright>
+//----------------------------------------------------------------------- 
 namespace ConsoleApplication1
 {
-    class Program
+    using System;
+
+    /// <summary>
+    /// Domyślna klasa programu konsolowego tworzona przez Monodevelop
+    /// </summary>
+    internal class Program
```

Comment:
NIP to dobra nazwa. SprawdzNIP to byłaby ewentualnie nazwa dla metody.

Swoją drogą, wg Microsoftu poprawna nazwa to Nip, bo jest to akronim trzyliterowy.
https://msdn.microsoft.com/en-us/library/ms229043%28v=vs.100%29.aspx